src/steps/github-action-step.ts

```diff
@@ -25,12 +33,14 @@ export interface GitHubActionStepProps {
 export class GitHubActionStep extends Step {
   public readonly env: Record<string, string>;
   public readonly jobSteps: JobStep[];
+  public readonly environment?: string;
```

Comment:
lets make this githubEnvironment too. essentially i want _us_ to call this thing githubEnvironment everywhere internally